manifests/init.pp

```diff
@@ -12,6 +12,8 @@
 #
 # $enable_file::                        Enable file content plugin
 #
+# $proxy_pulp_isos_to_pulpcore::        Proxy /pulp/isos to pulpcore at /pulp/content 
+#
```

Comment:
I'm thinking this should be in the advanced parameters section as we don't want users to mess with this typically.

Reply:
Makes sense, thanks